### ISSUE TITLE
When moving a piece, the editing/action menus should not pop up by default

### ISSUE THREAD

**Issue by dudefaceguy** [jrmi/airboardgame#264]
Whenever I move a piece, the action/editing menu pops up and blocks part of the board. I then have to click away from it before I can access the part of the board that the menu blocks. If I just want to play a game that requires moving pieces around, it is irritating to have this menu pop up every time I move a piece.

Not a critical bug, but just irritating when trying to actually play a game (especially my game, which frequently requires me to move 2 objects right next to each other, so the menu frequently hides the second object I have to move).

**Comment by jrmi** [comment]
I noticed that too, but I haven't found a good idea to keep the advantage of this behavior and improve usability.

When the menu is displayed on each selection :
- This helps new players to know what they can do
- This allows a fast work 

If it does not appear automatically, what action makes it appear? And how to make this action easily discoverable by newcomers?

If you have an idea...

**Comment by dudefaceguy** [comment]
I agree that it is better to have the menu appear automatically than to hide it somehow. I was envisioning that the menu always appears when simply clicking, but moving the object makes the menu disappear.

________________________________

**Comment by dudefaceguy** [comment]
I see that the menu already disappears while I am moving an item - I think it should not reappear after I finish the movement, unless I select the item again. This would make it easier to play the games that I personally play here, which involve primarily moving pieces without otherwise interacting with them. But, maybe not everyone is like me :)

________________________________

**Comment by jrmi** [comment]
> I see that the menu already disappears while I am moving an item - I think it should not reappear after I finish the movement, unless I select the item again. This would make it easier to play the games that I personally play here, which involve primarily moving pieces without otherwise interacting with them. But, maybe not everyone is like me :)

I'll try to implement this to see if it works.

**Comment by jrmi** [comment]
Actually I tried another idea: a new button allow users to hide menu on click if they want. Since hidden, menu still appears on mouse hover where it should be.

@dudefaceguy can you try the new version on https://airboardgame.net ?

**Comment by dudefaceguy** [comment]
It works perfectly so far - I've updated Mattock to include snapping. It works on desktop and mobile. It also mostly takes care of the problem of hiding objects with your finger on mobile, since accuracy isn't as important now.

The hide menu option is also great, and works on both mobile and desktop. Although, I think you have left the French cacher in the English version :) One aesthetic nitpick - the red selection box remains around the object I just touched. It looks cleaner without this box, but I understand if this is necessary for other games with more complex structures than the simple abstracts I am playing now. Not a big deal anyhow as it doesn't block the view.

You have implemented my top 3 items in just a few weeks, it's impressive! I can't thank you enough, this has everything to be an amazing tool now. I'm going to post to the abstract forums about it to try to get people interested in using it for testing their games. I may also add an Abstract Sandbox game with basic boards and pieces. Really exciting!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
